pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
     "gitpython>=3.0",
     "importlib-metadata>=3.6,<9.0",
     "importlib_resources>=1.3,<7.0",  # The `files()` API was introduced in `importlib_resources` 1.3 and Python 3.9.
+    "kedro-telemetry>=0.3.1",
```

Comment:
Is `0.3.1` an acceptable version or should we just up this to `0.5.0` so we get the latest with all improvements on identifying users and projects?

Reply:
IMO makes sense to update, indeed. @DimedS, wdyt?

Reply:
Yes, I believe it makes sense to update it now, but it's more important to remember to update it with a new kedro-telemetry version after the next release of kedro-telemetry. This should be done after we complete all opt-out work, but before the Kedro release.